src/main/java/com/msgilligan/bitcoin/BTC.java

```diff
@@ -8,7 +8,7 @@
  * Utility class for converting from satoshis (BigInteger) to bitcoins (BigDecimal) and vice-versa
  */
 public class BTC  {
-    public static final long satoshisPerBitcoin = 100000000;
+    public static final long satoshisPerBitcoin = 100000000L;
```

Comment:
It is, but I saw it somewhere else in the code and copied the style for consistency. :)